build 的时候报错: Can't resolve 'bg.png'

node v9.10.0
umi > 2.0.0-beta.19，用的是本地 link 的

执行 umi dev 运行正常，界面效果也正常
但是打包的时候报错了，错误信息如下：
```
error in ./src/pages/Login/index.less
ModuleNotFoundError: Module not found: Error: Can't resolve 'bg.png'
```
index.less 的代码如下：
```
.bg {
  background-image: url(./bg.png);
  background-position: 50%;
  background-repeat: no-repeat;
  background-size: cover;
  height: 100%;
}
```
.umirc.js 配置如下：
```
export default {
  publicPath: 'http://cdn.com/foo',
  plugins: [
    ['umi-plugin-react', {
      dva: true,
      antd: true,
      routes: {
        exclude: [/model.js/, /services.js/, /fields.js/],
      },
    }],
  ],
  extraBabelPlugins: [
    [require.resolve('babel-plugin-import'), { libraryName: 'carno', camel2DashComponentName: false }, 'carno'],
  ],
  chainWebpack(config, { webpack }) {
    config.plugin('provide').use(new webpack.ProvidePlugin({ React: 'react' }));
  },
  proxy: {
    '/api': {
      target: 'http://jsonplaceholder.typicode.com/',
      changeOrigin: true,
      pathRewrite: { '^/api' : '' }
    }
  },
};
```

--- a/packages/af-webpack/src/getConfig/css.js
+++ b/packages/af-webpack/src/getConfig/css.js
@@ -40,6 +40,8 @@ export default function(webpackConfig, opts) {
                 'default',
                 opts.cssnano || {
                   mergeRules: false,
+                  // ref: https://github.com/umijs/umi/issues/955
+                  normalizeUrl: false,
                 },
               ],
             }),
@@ -48,9 +50,11 @@ export default function(webpackConfig, opts) {
   };
   const cssModulesConfig = {
     modules: true,
-    localIdentName: cssOpts.localIdentName || (isDev
-      ? '[name]__[local]___[hash:base64:5]'
-      : '[local]___[hash:base64:5]'),
+    localIdentName:
+      cssOpts.localIdentName ||
+      (isDev
+        ? '[name]__[local]___[hash:base64:5]'
+        : '[local]___[hash:base64:5]'),
   };
   const lessOptions = {
     modifyVars: theme,

